library/ceph_config.py

```diff
@@ -135,15 +149,15 @@ def main() -> None:
     startd = datetime.datetime.now()
     changed = False
 
-    rc, cmd, out, err = get_or_set_option(module, 'get', who, option, value)
-    if rc:
-        fatal(message=f"Can't get current value. who={who} option={option}", module=module)
+    rc, cmd, out, err = get_config_dump(module)
```

Comment:
before this change when the action was `get`, `rc, cmd, out, err` would have ended up set as the result of the `ceph config get` command. With this change, it's now from the result of the `ceph config dump`. Are we doing anything with that out that means that would be an issue? I.e. if we were returning that output to show the user the value of the config option they'd now get a big json blob of all of the options.

Reply:
it shouldn't be an issue, `out` is set L158 in the case where the value is already set or L160 if a new value had to be set.

Reply:
true